validations/semantic_artifacts_test.go

```diff
@@ -69,14 +67,35 @@ resources:
       service: destination
    type: org.cloudfoundry.managed-service
 `)
+
+	It("ifModulePathExists", func() {
```

Comment:
In each of the tests check the number of issues is as expected

Reply:
done